employee&managment/models.js

```diff
@@ -0,0 +1,39 @@
+// models.js
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('./db');
+
+const Department = sequelize.define('Department', {
+  name: {
+    type: DataTypes.STRING(100),
+    allowNull: false,
+  },
+});
+
+const Employee = sequelize.define('Employee', {
+  firstName: {
+    type: DataTypes.STRING(100),
+    allowNull: false,
+  },
+  lastName: {
+    type: DataTypes.STRING(100),
+    allowNull: false,
+  },
+  salary: {
+    type: DataTypes.FLOAT,
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider decimal type for salary field</b></div>
<div id="fix">

Consider using `DataTypes.DECIMAL` instead of `DataTypes.FLOAT` for the salary field to avoid floating-point precision issues with monetary values
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -21,7 +21,7 @@
    salary: {
 -    type: DataTypes.FLOAT,
 +    type: DataTypes.DECIMAL(10,2),
      allowNull: false,
    },
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/rishabhbohra3/Bito-CRA-Issues-Dataset/pull/3#issuecomment-2513466440>#82d15d</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged